src/docdb/nodes.ts

```diff
@@ -67,13 +71,17 @@ export class DocDBDatabaseNode implements INode {
 
 }
 
-
 export class DocDBCollectionNode implements INode {
 
 	constructor(readonly id: string, readonly db: DocDBDatabaseNode) {
 	}
 
 	readonly contextValue: string = "cosmosDBDocumentCollection";
+	private _children = [];
+	private _hasFetched: boolean = false;
+	private _loadMoreNode: LoadMoreNode = null;
+	private _hasMore: boolean;
+	private iterator = null;
```

Comment:
Don't set these values to null and make sure to define a type. For example, this should be:
```
private iterator: QueryIterator<any>;
```